src/components/Header/Header.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const header = () => {
```

Comment:
I'd recommend to name React Components with first letter being upper cased and not use the default import.